src/main/window-manager.ts

```diff
@@ -15,6 +15,9 @@ export class WindowManager extends Singleton {
   protected windowState: windowStateKeeper.State;
   protected disposers: Record<string, Function> = {};
 
+  @observable mainViewInitiallyLoaded = false;
+  whenLoaded = when(() => this.mainViewInitiallyLoaded);
```

Comment:
Yup, this is needed to make sure that the `renderer` has loaded before broadcasting to the renderer because before this is set the renderer is not guaranteed to be in existence.

Sure I can add that as a comment.